doc/dev.md

```diff
@@ -24,6 +24,7 @@ The SQL LRS can be built or run with the following Makefile targets. They can be
 | `ci`               | Called when running continuous integration; runs all test cases in all SQL flavors.                                                                                                                          |
 | `test-sqlite`      | Run all tests with SQLite database.                                                                                                                                                                          |
 | `test-mariadb`       | Run all tests with Mariadb database version 11.7.2. Set the `LRSQL_TEST_DB_VERSION` env var to a valid Postgres docker tag to use another version.                                                              |
+| `test-mysql`       | Run all tests with MySQL database version 8.0.44.                                                                                                                                                             |
```

Comment:
probably should include a full accounting for make actions on maria and mysql like we do for postgresql